### ISSUE TITLE
GBP (£) incorrectly placed

### ISSUE THREAD

**Issue by Bourne-ID** [onyxbits/TradeTrax#1]
The GBP symbol (£) appears at the beginning of the currency (£1.23) rather than the end - the app displays the GBP symbol at the end. (EG summary.tml variable ${totalProfit} doesn't cater for varying locations, which could be fixed by DecimalFormat.getCurrencyInstance().format(xxxx) ).

**Comment by onyxbits** [comment]
Yeah, known issue. Same problem with dollars as well. The problem is that DecimalFormat.getCurrencyInstance() is not suitable for TradeTrax as it is suppose to also support non ISO 4217 currencies (e.g. bitcoins or MMORPG money). Sadly, there seems to be no way to dynamically add new currencies to the JRE.

What would need to be fixed is the MoneyRepresentation class.
